assets/js/blocks/product-collection/inspector-controls/index.tsx

```diff
@@ -43,6 +44,9 @@ const ProductCollectionInspectorControls = (
 
 	return (
 		<InspectorControls>
+			<BlockControls>
+				<DisplayLayoutControl { ...props } />
```

Comment:
Nitpicking: I was thinking about this for other places too, where we are passing all props to the control. Passing all properties to a child component using `{ ...props }` is a quick and easy way to pass properties down, but it can potentially lead to unnecessary prop-drilling and make it harder to track where certain properties are coming from or being used. It can also result in unnecessary re-rendering if the parent component has other props that change frequently but are not used by `DisplayLayoutControl`.

What do you think? 🙂

Reply:
I agree. I used `{ ...props }` to be aligned with the rest of a file, but I agree in principle, so I changed that in [this commit](https://github.com/woocommerce/woocommerce-blocks/pull/10145/commits/c196069795dd46469e655baee57a6cf221bd1bca). I'll make another PR updating this for the rest of Inspector Controls!

Reply:
Sounds great! Thanks for making the changes 🙌🏻